Bug: `filter_paths` does not filter out `unindexed-event-address` findings

### Describe the issue:

Slither reports an `unindexed-event-address` finding (that got introduced recently in #2866) for files in paths that should be filtered out.


[`slither.config.json`](https://github.com/anoma/pa-evm/blob/fb3fb2076058ad8a3dc1036213d5885e075f4f07/contracts/slither.config.json)
```json
{
  "filter_paths": "dependencies/",
  "detectors_to_exclude": [
    "pragma",
    "naming-convention",
    "cyclomatic-complexity"
  ]
}
```
The failing workflow using the recent slither version `0.11.4`:
https://github.com/anoma/pa-evm/actions/runs/21061122647/job/60567464171



### Code example to reproduce the issue:

1. `git clone https://github.com/anoma/pa-evm`
2. `cd ./pa-evm/contracts`
3. `forge soldeer install && forge build`
4. `slither .`

### Version:

0.11.4

### Relevant log output:

```shell
slither .

'forge clean' running (wd: /Users/runner/work/pa-evm/pa-evm/contracts)
'forge config --json' running
'forge build --build-info --skip ./test/** ./script/** --force' running (wd: /Users/runner/work/pa-evm/pa-evm/contracts)
INFO:Detectors:
Detector: unindexed-event-address
Event Pausable.Paused(address) (dependencies/@openzeppelin-contracts-5.5.0/utils/Pausable.sol#23) has address parameters but no indexed parameters
Event Pausable.Unpaused(address) (dependencies/@openzeppelin-contracts-5.5.0/utils/Pausable.sol#28) has address parameters but no indexed parameters
Reference: https://github.com/crytic/slither/wiki/Detector-Documentation#unindexed-event-address-parameters
INFO:Slither:. analyzed (43 contracts with 97 detectors), 2 result(s) found
Error: Process completed with exit code 255.
```

--- a/slither/detectors/attributes/unindexed_event_address.py
+++ b/slither/detectors/attributes/unindexed_event_address.py
@@ -50,8 +50,6 @@ Off-chain tools cannot efficiently filter transfers by sender or recipient addre
         "to enable efficient off-chain filtering."
     )
 
-    STANDARD_JSON = False
-
     @staticmethod
     def _has_unindexed_address(event: Event) -> bool:
         """

